From cc45cbb0d619cd9ad9d7e7724733434a2066578e Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Wed, 4 Nov 2020 12:55:21 +0100
Subject: [PATCH 1/1] doc: qemu-riscv: describe attaching a disk image

How to add an emulated disk to QEMU may not be known to the user.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
cf.
riscv: enable SATA disk on QEMU RISC-V
https://patchwork.ozlabs.org/project/uboot/list/?series=212093
---
 doc/board/emulation/qemu-riscv.rst | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/doc/board/emulation/qemu-riscv.rst b/doc/board/emulation/qemu-riscv.rst
index b68db95a45..3acd40630e 100644
--- a/doc/board/emulation/qemu-riscv.rst
+++ b/doc/board/emulation/qemu-riscv.rst
@@ -105,3 +105,11 @@ configurations are:
 
     qemu-system-riscv64 -nographic -machine virt -bios spl/u-boot-spl \
     -device loader,file=u-boot.itb,addr=0x80200000
+
+An attached disk can be emulated by adding::
+
+    -device ich9-ahci,id=ahci \
+    -drive if=none,file=riscv64.img,format=raw,id=mydisk \
+    -device ide-hd,drive=mydisk,bus=ahci.0
+
+You will have to run 'scsi scan' to use it.
-- 
2.28.0

